### ISSUE TITLE
Programming languages

### ISSUE THREAD

**Issue by Soveu** [mos-dev-wiki/meta#5]
Do we want to only include Rust (with some asm! snippets) or do we also want C/C++ or other languages?

**Comment by AbleTheAbove** [comment]
I think as most/all of us are from the rust discord it would be easier on us if we focused on rust and asm at the moment and if there is need to expand we can do so!

**Comment by Evrey** [comment]
Yeah, we can *prepare* articles for tabs around code blocks, so new snippets in other proglangs can be added. But ideally we'd be just focussing on memory-mapped register addresses, hardware registers, offsets, all independent of actual code. *Especially* because many registers contain tightly packed bitfields, and most proglangs don't have these. (Arguably, Zig would be the better language for examples.)

**Comment by Stupremee** [comment]
I think it would be relatively easy and very helpful to provide examples in multiple languages. Zig, C, and Rust for example, and then have a small selector on every codeblock

**Comment by Evrey** [comment]
Btw., there should also be a strategy to tell our readers how to report mistakes in articles or provide additional info. Possibly how PhilOpp does it with his blog: Have a readily available issue per article.
